src/stylesheets/components/_sidenav.scss

```diff
@@ -1,8 +1,28 @@
 
 .usa-sidenav-list {
   @include usa-sidenav-list;
+  @include margin(null (-$site-margins-mobile) null (-$site-margins-mobile));
```

Comment:
The negative left and right margins are contextual related to the space of it's parent container, specifically it's dependent on being inside a `usa-grid` that provides the margins. For this reason, I think it may be safer to make this relationship explicit in the code, i.e. `.usa-grid .usa-sidenav-list` for the negative margins. What if a user isn't using our grid?

Reply:
good point. will do.